### PR TITLE
Update contributor's documentation [OPEN-603]

### PR DESCRIPTION
**Pull request by sarina**
@mhoeber could you review this please, or suggest who should?

### CODE REVIEW

File: en_us/developers/source/process/overview.rst [hunk 1]
```diff
@@ -15,6 +15,81 @@ and other contributors can expect from each other. The goals are:
...
   as much as possible.
 
+Overview
+--------
+
+Get in touch with us right now! We want to talk to you about your work as early
+as possible in your development cycle, ideally when you start designing your
+feature. This will help you so much, we promise: we'll help guide your work so
+you don't have to rearchitect it late in your development cycle. We'll help you
+understand the analytics, performance, test, and other various requirements up
```
Comment (mhoeber) [inline en_us/developers/source/process/overview.rst]: I'd add accessibility

File: en_us/developers/source/process/overview.rst [hunk 2]
```diff
@@ -15,6 +15,81 @@ and other contributors can expect from each other. The goals are:
...
+
+You can get in touch with us using JIRA, IRC, or mailing lists. `The
+instructions here`_ explain how to use each tool; use whichever you prefer.
+
+You should get in touch with us regarding any work you wish to contribute
+upstream, including but not limited to:
+
+* Core platform changes (changes to the edx-platform repo)
+* Changes to any associated repos (edx-analytics, configuration, XBlock, etc)
+* A new XBlock you're writing that you wish to have run on edx.org
```
Comment (mhoeber) [inline en_us/developers/source/process/overview.rst]: wish -> intend

File: en_us/developers/source/process/overview.rst [hunk 3]
```diff
@@ -15,6 +15,81 @@ and other contributors can expect from each other. The goals are:
...
+
+We do not need to review your code if you are writing:
+
+* An XBlock to be run on your own server, not intended to run on edx.org
+* LTI units that are running on your own server
```
Comment (mhoeber) [inline en_us/developers/source/process/overview.rst]: is "units" a standard LTI term?

**Comment by mhoeber** [comment]
:+1:

File: en_us/developers/source/process/overview.rst (hunk 3 above)
Comment (sarina) [inline en_us/developers/source/process/overview.rst]: Apparently not. I wll change to "tool" which seems to be more common.

**Pull request merged**